eurec4a_environment/variables/calc_density.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import xarray as xr
+from ...constants import Rd, Rv, eps
```

Comment:
```suggestion
from ..constants import Rd, Rv, eps
```
This should solve the pytest error, I think. Not sure.

Reply:
Yes I think you're right :) 

```
eurec4a_environment/variables/calc_density.py:6: in <module>
    from ...constants import Rd, Rv, eps
E   ValueError: attempted relative import beyond top-level package
```